eknrenderer/Makefile.am

```diff
@@ -0,0 +1,143 @@
+## Process this file with automake to produce Makefile.in
+
+# Copyright 2018 Endless Mobile, Inc.
+
+## -----------
+## Makefile.am
+## -----------
+## Please keep this file well-commented.
+
+# Subdirectories where we also want to run make
+# (Make sure that the library is built before the docs)
+SUBDIRS = .
+
+# Our Autoconf macros are kept in the m4/ directory
+ACLOCAL_AMFLAGS = -I m4
+
+# Forward declare variables we'll be modifying all over
+CLEANFILES =
+DISTCLEANFILES =
+BUILT_SOURCES =
+EXTRA_DIST = README.md eos-knowledge-renderer.pc.in
+lib_LTLIBRARIES =
+
+@CODE_COVERAGE_RULES@
+
+# # # RENDERER LIBRARY # # #
+
+eknrenderer_public_installed_headers = eknrenderer/eos-knowledge-renderer.h
+eknrenderer_private_installed_headers = \
+	eknrenderer/eknr-boxed-set-info.h \
+	eknrenderer/eknr-errors.h \
+	eknrenderer/eknr-renderer.h \
+	$(NULL)
+eknrenderer_sources = \
+	eknrenderer/eknr-boxed-set-info.c \
+	eknrenderer/eknr-errors.c \
+	eknrenderer/eknr-renderer.c \
+	$(NULL)
+
+glib_enum_headers = \
+	$(eknrenderer_public_installed_headers) \
+	$(NULL)
+glib_enum_h = eknrenderer/eknr-enums.h
+glib_enum_c = eknrenderer/eknr-enums.c
+include $(top_srcdir)/Makefile.am.enums
+
+lib_LTLIBRARIES += libeos-knowledge-renderer-@EKN_RENDERER_API_VERSION@.la
+libeos_knowledge_renderer_@EKN_RENDERER_API_VERSION@_la_SOURCES = \
+	$(glib_enum_h) \
+	$(glib_enum_c) \
+	$(eknrenderer_public_installed_headers) \
+	$(eknrenderer_sources) \
+	$(NULL)
+
+libeos_knowledge_renderer_@EKN_RENDERER_API_VERSION@_la_CPPFLAGS = \
+	@EKN_RENDERER_CFLAGS@ \
+	$(CODE_COVERAGE_CPPFLAGS) \
+	-DG_LOG_DOMAIN=\"EosKnowledgeRenderer\" \
+	-DCOMPILING_EKN_RENDERER \
+	-DMATHJAX_PATH=\"@MATHJAXDIR@\"
```

Comment:
ekncontent uses a `config.h.in` for this. I couldn't get the variable substitution there to work right, so I've just opted for doing this for now.